doc/release-notes/5.0-release-notes.md

```diff
@@ -0,0 +1,321 @@
+
+We've included this in the step-by-step instructions below.
+
+## Notes for Tool Developers and Integrators
+
+## Complete List of Changes
+
+For the complete list of code changes in this release, see the [5.0 Milestone](https://github.com/IQSS/dataverse/milestone/89?closed=1) in Github.
+
+For help with upgrading, installing, or general questions please post to the [Dataverse Google Group](https://groups.google.com/forum/#!forum/dataverse-community) or email support@dataverse.org.
+
+## Installation
+
+If this is a new installation, please see our [Installation Guide](http://guides.dataverse.org/en/5.0/installation/)
+
+## Upgrade Dataverse from Glassfish 4.1 to Payara 5
+
+The instructions below describe the upgrade procedure based on moving an existing glassfish4 domain directory under Payara. We recommend this method instead of setting up a brand-new Payara domain using the installer because it appears to be the easiest way to recreate your current configuration and preserve all your data.
+
+1. Download Payara, v5.2020.2 as of this writing:
+
+   `curl -L -O https://github.com/payara/Payara/releases/download/payara-server-5.2020.2/payara-5.2020.2.zip`
+   `sha256sum payara-5.2020.2.zip`
+      1f5f7ea30901b1b4c7bcdfa5591881a700c9b7e2022ae3894192ba97eb83cc3e
+
+2. Unzip it somewhere (/usr/local is a safe bet)
+
+   `sudo unzip payara-5.2020.2.zip -d /usr/local/`
+
+3. Copy the Postgres driver to /usr/local/payara5/glassfish/lib
+
+   `sudo cp /usr/local/glassfish4/glassfish/lib/postgresql-42.2.9.jar /usr/local/payara5/glassfish/lib/`
+
+4. Move payara5/glassfish/domains/domain1 out of the way
+
+   `sudo mv /usr/local/payara5/glassfish/domains/domain1 /usr/local/payara5/glassfish/domains/domain1.orig`
+
+5. Undeploy the Dataverse web application (if deployed; version 4.20 is assumed in the example below)
+
+   `sudo /usr/local/glassfish4/bin/asadmin list-applications`
+   `sudo /usr/local/glassfish4/bin/asadmin undeploy dataverse-4.20`
+
+6. Stop Glassfish; copy domain1 to Payara
+
+   `sudo /usr/local/glassfish4/bin/asadmin stop-domain`
+   `sudo cp -ar /usr/local/glassfish4/glassfish/domains/domain1 /usr/local/payara5/glassfish/domains/`
+
+7. Remove the Glassfish cache directories
+
+   `sudo rm -rf /usr/local/payara5/glassfish/domains/domain1/generated/`  `sudo rm -rf /usr/local/payara5/glassfish/domains/domain1/osgi-cache/`
+
+8. In domain.xml:
+
+ Replace the -XX:PermSize and -XX:MaxPermSize JVM options with -XX:MetaspaceSize and -XX:MaxMetaspaceSize.
+
+   <jvm-options>-XX:MetaspaceSize=256m</jvm-options>
+   <jvm-options>-XX:MaxMetaspaceSize=512m</jvm-options>
+
+Set both Xmx and Xms at startup to avoid runtime re-allocation. Your Xmx value should likely be higher:
+
+   <jvm-options>-Xmx2048m</jvm-options>
+   <jvm-options>-Xms2048m</jvm-options>
```

Comment:
Not sure why it was needed, mentioning the Xmx adn Xms lines to the upgrade instruction. I mean, it's an existing installation - they must have these settings in their config already. I can see how this can confuse somebody. 
I would simply remove this paragraph and the last two jvm-options lines.

Reply:
@landreev I removed this in https://github.com/IQSS/dataverse/commit/c593f89bff1e4ba3e5988bc9dd81708ff739f0a1

heads up to @donsizemore in case this was included for some other reason we're not seeing